lenses/tests/test_krb5.aug

```diff
@@ -1071,3 +1071,37 @@ test Krb5.lns get dbmodules_test =
     }
     { "db_module_dir" = "/some/path" }
   }
+
+(* [plugins] test *)
+let plugins_test = "[plugins]
+    clpreauth = {
+        module = mypreauth:/path/to/mypreauth.so
+    }
+    ccselect = {
+        disable = k5identity
+    }
+    pwqual = {
+        module = mymodule:/path/to/mymodule.so
+        module = mymodule2:/path/to/mymodule2.so
+        enable_only = mymodule
+    }
+    kadm5_hook = {
+    }
+"
+
+test Krb5.lns get plugins_test =
+  { "plugins"
+    { "clpreauth"
+      { "module" = "mypreauth:/path/to/mypreauth.so" }
+    }
+    { "ccselect"
+      { "disable" = "k5identity" }
+    }
+    { "pwqual"
+      { "module" = "mymodule:/path/to/mymodule.so" }
```

Comment:
Would this be easier to work with as a deeper tree where `module` is a `seq` with `mymodule` = `/path/to/mymodule.so` so folks who need to easily switch from `/usr/lib` to `/usr/lib64` for a specific module can target more directly?

Reply:
I'm not sure how `seq` would help in the case you mentioned, since you can use the following expressions to find your node:
- `/files/etc/krb5.conf/plugins/*/module[. =~ regexp(".*/mymodule2.*")]`
- `/files/etc/krb5.conf/plugins/*/module[. =~ regexp("mymodule:.*")]`

Reply:
I'd avoid `seq` as much as possible when it's not necessary to use it. Imo `seq` is only useful when several consecutive nodes have the same label and you want to make sure they don't exchange indent/spaces when one of them is removed.